.circleci/config.yml

```diff
@@ -98,11 +98,12 @@ jobs:
           # NOTE: -Werror disabled in CI, as we currently have >4k warnings.
           cd build \
             && cmake .. -DCMAKE_BUILD_TYPE=Debug -DENABLE_COVERAGE=On -DCPACK_GENERATOR=DEB \
-                -DENABLE_COMPILER_WARNINGS=On -DENABLE_WERROR=Off -DCMAKE_EXPORT_COMPILE_COMMANDS=On
+                -DENABLE_COMPILER_WARNINGS=On -DENABLE_WERROR=Off -DCMAKE_EXPORT_COMPILE_COMMANDS=On \
+                -DENABLE_ADDRESS_SANITIZER=ON -DENABLE_PYTHON_BINDINGS=Off
```

Comment:
@SvetlanaBayarovich is it true that the build fails with `ENABLE_UNDEFINED_SANITIZER=On`? because of https://github.com/valhalla/valhalla/pull/2296#issuecomment-608207370

Reply:
and i guess address sanitizer finds something in the python bindings as well?

Reply:
@kevinkreiser The build fails with `leak sanitizer` (which is a part of ASan) enabled with what looks very similar to the comment you shared. And that is quite a curious finding of yours 🤔 I planned to investigate it separately and enable "pure" ASan on CI for now. This leak snaitizer is not well supported on all the platforms and is actually unavailable for my Apple's `clang` which is really irritating

Reply:
As for python bindings, the problem is entirely different - it is about link order of libraries:
```
==22382==ASan runtime does not come first in initial library list; you should either link runtime to your application or manually preload it with LD_PRELOAD.
test/CMakeFiles/run-python_valhalla.dir/build.make:62: recipe for target 'test/python_valhalla.log' failed
```
Does it actually make sense to test python bindings?

Reply:
it does make sense to test the python bindings. many collaborators are making use of them. i suppose that its ok to remove from debug so long as we dont remove from the other builds